package.json

```diff
@@ -316,7 +316,7 @@
     "electron-builder": "26.0.19",
     "eslint": "^9.36.0",
     "eslint-config-airbnb-typescript": "^18.0.0",
-    "eslint-config-expensify": "2.0.100",
+    "eslint-config-expensify": "github:Expensify/eslint-config-expensify#0432ea0ed4da293811502d6aabc60190d24ad1ad",
```

Comment:
Why'd you change this to a GitHub ref? Was there some problem with the npm-published version?

Reply:
That was for testing of the total lint time on the PR 🙌